lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,16 @@
 
 def palindrome_permutation?(string)
-  
+  hash_table = {}
+
+  string.each_char do |c|
+    hash_table[c] ? hash_table[c] += 1 : hash_table[c] = 1
```

Comment:
I think the ternary operator makes it a little harder to scan. Even though it is more lines of code, I think the following would be a quicker read for most folks:
```
if hash_table[c]
  hash_table[c] += 1
else
  hash_table[c] = 1
end
```